lib/web_ui/lib/src/engine/surface/scene_builder.dart

```diff
@@ -97,6 +97,14 @@ class SurfaceSceneBuilder implements ui.SceneBuilder {
     if (matrix4.length != 16) {
       throw ArgumentError('"matrix4" must have 16 entries.');
     }
+    if (_surfaceStack.length == 1) {
+      // Top level transform contains view configuration to scale
+      // scene to devicepixelratio. Use identity instead since web browser
```

Comment:
nit: to be more specific CSS uses logical pixels (e.g. WebGL doesn't).

Reply:
Done.